### ISSUE TITLE
Organization app submission

### ISSUE THREAD

**Issue by ghost** [elementary/houston#644]
I have no way to submit an app to AppCenter on https://developer.elementary.io/dashboard , the dashboard only shows the personal repositories, and not the organizations, who i'm the owner, repositories.

**Comment by lewisgoddard** [comment]
On the [developer dashboard](https://developer.elementary.io/dashboard) click **Manage Apps…** (top left), followed by the **Configure** button (top right), and you should be able to expand your AppCenter integration on GitHub to any organisation you are a member of.

**Comment by lewisgoddard** [comment]
@btkostner Maybe we should take this as a UX hint?

**Comment by eudaldgr** [comment]
This is not closed. i do this instructions and still don't work in my case, i'm not stupid. i attach some screenshots. 
![1](https://user-images.githubusercontent.com/30153954/45429126-2d856f00-b692-11e8-923e-81e62394a859.png)
![2](https://user-images.githubusercontent.com/30153954/45429127-2d856f00-b692-11e8-8313-84996a17b583.png)
![3](https://user-images.githubusercontent.com/30153954/45429128-2e1e0580-b692-11e8-89e9-54fbc2f99ec8.png)
![4](https://user-images.githubusercontent.com/30153954/45429129-2eb69c00-b692-11e8-99ce-513aed2712c1.png)

**Comment by eudaldgr** [comment]
I have done what you said several times before posting the issue

**Comment by btkostner** [comment]
@eudaldgr log out and back in

**Comment by eudaldgr** [comment]
i have do it several times before posting this, log out from the dashboard, from github, even from the computer (joke), and nothing... still not working. Deleted the github app permision, re-joined in again the permisions, on diferents days, at night, at morning, creating diferent organizations, and never work. never. i don't know if it's houstonCI or maybe my account, but never works with me

**Comment by eudaldgr** [comment]
And this issue it's not closed, i don't know why someone close this without knowing what happens before, but it's not fair

**Comment by eudaldgr** [comment]
@lewisgoddard this isn't a UX hint

**Comment by btkostner** [comment]
I'll take a look at the logs tonight when I get home and report back

**Comment by btkostner** [comment]
Sorry for the delay. I looked in the logs and it seems that elements was added to the database, so the problem is probably between GitHub and houston identifing what repos you have access to.

In the mean time, if you want to publish your app, I can manually press the button for you.

**Comment by eudaldgr** [comment]
yeah! press the button for me, but if you can, delete the other published app with elements name. i changed the ownership of the repository, and there will need only one elements app on appcenter. thanks

**Comment by eudaldgr** [comment]
I have added a new repo to the organization account, and seems to be not displaying the new one also, maybe this isn't for the elements repo. thank you very much!

**Comment by btkostner** [comment]
@eudaldgr now that I think about it, I think @nathandyer had a similar issue when we were in beta. If I remember correctly it was because of GitHub and the way organization visibility worked. Can you try making yourself visible on the organization, log out of houston, and log back in.

https://github.com/orgs/FreeeRadicals/people

**Comment by mirkobrombin** [comment]
Maybe can help.
I created a second account, linked to the organization, granted the permissions to the AppCenter and now I see the project in both. 🎉️

**Comment by miguelsotobaez** [comment]
@btkostner I fix this issue by revoke appcenter apps in my github account & organization and added again. Hope this help the team to make a root fix.

Regards.
